src/components/Banner/SpringSurgeBanner.tsx

```diff
@@ -25,7 +25,8 @@ const BannerInner: React.FC = () => {
   return (
     <InnerContainer>
       <Body>
-        Hospitalizations are rising in {SeveralStatesLink}. Help prevent another
+        Daily new cases are flat or rising in two-thirds of states and
```

Comment:
FWIW- 32 states have r(t) of 1.01 or higher, so you could probably just say "are rising". I'm fine either way though.

Reply:
1.01 is within the noise for increasing in my mind, especially given the vagaries in reporting recently. You could make the case that 0.98 should be included in "essentially flat" too, which is why I kept it at two-thirds instead of 32 states. It also makes it more evergreen without losing much messaging.